code/datums/wires/wires.dm

```diff
@@ -123,8 +123,9 @@ var/list/wireColours = list("red", "blue", "green", "darkred", "orange", "brown"
 	for(var/colour in wires)
 		html += "<tr>"
 		var/datum/wire_description/wd = get_description(GetIndex(colour))
-		if(user.stats && user.stats.getPerk(PERK_TECHNOMANCER) || user_skill && (wd.skill_level <= user_skill))
-			html += "<td[row_options1]><font color='[colour]'>[wd.description]</font></td>"
+		if(wd)
+			if(user.stats && user.stats.getPerk(PERK_TECHNOMANCER) || user_skill && (wd.skill_level <= user_skill))
+				html += "<td[row_options1]><font color='[colour]'>[wd.description]</font></td>"
```

Comment:
```suggestion
		if(wd && user.stats && user.stats.getPerk(PERK_TECHNOMANCER) || user_skill && (wd.skill_level <= user_skill))
			html += "<td[row_options1]><font color='[colour]'>[wd.description]</font></td>"
```

Reply:
this will still runtime as "cannot read null.skill_level"

Reply:
Could try `wd?.skill_level`, but whatever, PR is good as is.

Reply:
`wd?.skill_level` also runtimes, byond moment I guess